tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -94,21 +95,21 @@ def vmobj_to_list(o):
 
 def run_tvm_graph(graph_def, input_data, input_node, num_output=1,
                   target='llvm', out_names=None, opt_level=3, mode='graph_runtime',
-                  cuda_layout="NCHW"):
+                  layout=None, disabled_pass=None):
```

Comment:
It seems disabled_pass is not specified by any caller

Reply:
We might need to disable fold_scale_axis for ssd testing. https://discuss.tvm.ai/t/relay-pass-do-we-want-to-allow-letnode-in-foldscaleaxis-pass/6420